docs/sphinx/contributing/documentation.rst

```diff
@@ -17,11 +17,44 @@ reference using |doxygen|_ and |breathe|_.
 We always welcome suggestions on how to improve our documentation, as well as
 pull requests with corrections and additions.
 
+Prerequisites
+=============
+
+To build the |hpx| documentation, you need recent versions of the following
+packages:
+
+- ``python3``
+- ``sphinx`` (Python package)
```

Comment:
Can you add `3.5.4` after sphinx? as it's the version that as been pinned in the Dockerfile (as we also specify it for breathe)